### ISSUE TITLE
Update Append modal text

### ISSUE THREAD

**Issue by jonpmarino** [MapStory/mapstory-meta#1321]
- [X] ADD DATA: "Instead of adding edits to data one-by-one, with Append you can bulk upload a set of edits all at once. To Append, download a blank file below, add your features to it, save, and upload here."
- [X] UPLOAD YOUR FEATURE EDITS: "Upload the blank file you downloaded earlier that now has your new feature edits added. Once your append is complete, you should see all of your new features added as edits to this StoryLayer. 
- [x] Replace "Ok, were ready to create your StoryLayer", to "Ok, we're ready to update your StoryLayer"

**Comment by Coop56** [comment]
No longer relevant, this modal has been rebuilt.  Closing.
